src/sentry/templates/sentry/vercel-ui-hook.vercel

```diff
@@ -0,0 +1,9 @@
+<Page>
+{% block main %}
+    <Box display="flex" justifyContent="space-between">
+      <P>
+        • <Link href="https://www.sentry.io/settings/{{ org }}/integrations/vercel/{{ integration_id }}/">Add to project for {{ org }}</Link><BR />
```

Comment:
@ceorourke I just noticed the `•` symbol. Is this intentional?

Reply:
@scefali yeah is that bad? I just thought it looked a little more intentional than a link floating

Reply:
Yea, that's fine. Just curious :)